### PR TITLE
[react-native-extra-dimensions-android] types

### PR DESCRIPTION
**Pull request by kirusick**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

File: types/react-native-extra-dimensions-android/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-native-extra-dimensions-android 1.2
+// Project: https://github.com/Sunhat/react-native-extra-dimensions-android
+// Definitions by: Kiryl Arlouski <https://github.com/kirusick>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
```
Comment (peterblazejewicz) [inline types/react-native-extra-dimensions-android/index.d.ts:5]:
```suggestion
// TypeScript Version: 3.0
```
why do you need 3.6? The RN depends on 3.0 AFAIK

Comment (kirusick) [reply types/react-native-extra-dimensions-android/index.d.ts]: I didn't know this fact. Fixed

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@kirusick thx!

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by kirusick** [comment]
Ready to merge

**Pull request merged**